vmdb/lib/miq_automation_engine/service_models/miq_ae_service_auth_key_pair_openstack.rb

```diff
@@ -1,4 +1,4 @@
 module MiqAeMethodService
-  class MiqAeServiceAuthKeyPairOpenstack < MiqAeServiceAuthKeyPairCloud
+  class MiqAeServiceAuthKeyPairOpenstack < MiqAeServiceManageIQ_Providers_CloudManager_AuthKeyPair
```

Comment:
Yeah... the underscores are ugly, but seemed the simplest solution. They can't be nested, because of how their autoloads are defined.